### PR TITLE
Added Injuries & in Macros, Fixes

### PR DESCRIPTION
**Pull request by LeFrique**
+Added Injuries as Effects
+ Added Injury Effects to Schwere Verletzungen Macro
+ Fixed Pagedescription and Operations in Spell Effects
+ Fixed Images in Combat Conditions (again)

### CODE REVIEW

**Review by Byroks — COMMENTED** [review]
The skills are not selected in the templates. We might need to rework how skills are applied to the character or how they are structured so we can actually apply the the modifier without having to re-select the skill.

Encounterd a bug where negative modifiers from stats don't get applied to Abwehr and Angriffsbonus, will tackle that in a different Issue. (Fixed in #29)

File: packs/items/zauberwirkungen/Segnen.json
[the remark this answers is not in the document]
Comment (Byroks) [reply packs/items/zauberwirkungen/Segnen.json:1]: Should we merge this PR and you tackle the search in a different Issue or do you want to move this issue pack to in progress?

[the remark this answers is not in the document]
Comment (LeFrique) [reply packs/items/zauberwirkungen/Segnen.json:1]: I don't know what's better, I'd like to make an overhaul of all effects & macros when they are all in and have more time at the weekend for tackling the search. So when we merge this it's only some injuries which are affected think. The most skill using items are already in the spell effects I believe, so it doesn't harm the injuries quite much.

[the remark this answers is not in the document]
Comment (Byroks) [reply packs/items/zauberwirkungen/Segnen.json:1]: I just created a new release, so let's merge this and fix it till next release (next week sometime)

**Pull request merged**